docs/ide-integrations/visual-studio/settings-options.md

```diff
@@ -0,0 +1,18 @@
+
+# Extension Settings/Options
```

Comment:
This content seems to be obsolete/overlapping. The embedded links are 404. 

I think that the file content of `reqnroll_json_ide_section_description.md` should be in this file with a note that the extension settings can be controlled via the "ide" section of the config file.

Reply:
Combined the two pages.